plugin/pulp/plugin/__init__.py

```diff
@@ -0,0 +1,6 @@
+from pulp.platform import ProgressBar, ProgressSpinner, Repository, RepositoryContent
+
+from .cataloger import Cataloger
+from .distributor import RepositoryDistributor
+from .importer import RepositoryImporter
+from .profiler import Profiler
```

Comment:
I think it' ok to use them. These modules should always be relative to this `__init__.py` file. Even a few weeks ago I was arguing against not using them. Recently I re-evaluated my thoughts on them and now I think they are OK. Django uses them in its examples so they can't be that bad. The issue we had in pulp 2.y was that you couldn't run only one test. There was something about the way the tests were setup, run, or the test runner which would cause a line like this to fail:

`from ....base import PulpServerTests`

I think that was due to some other issue and wasn't related to it being a relative import necessarily.

I plan to leave them here for now and also leave them in in [pulp.platform.models](https://github.com/pulp/pulp/blob/6bb793295b65d041a7e10820d67e7c9eaa560865/platform/pulp/platform/models/__init__.py). If you want to change them I'm OK with that too.